tetramer_validator/__init__.py

```diff
@@ -0,0 +1,4 @@
+import argparse
```

Comment:
I don't like putting shared imports here. Each `.py` file should list its own imports -- it's much more clear.

Reply:
Changed import statements to match this.  https://github.com/IEDB/tetramer-validator/pull/17/commits/aed27624f28c69115926be90267d39ae2bf00ddf